concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2331,6 +2332,38 @@ jobs:
                   cf blue-green-deploy paas-metrics
                   cf delete -f paas-metrics-old
 
+        - task: deploy-paas-uaa-assets
+          config:
+            platform: linux
+            image_resource:
+              type: docker-image
+              source:
+                repository: governmentpaas/cf-cli
+                tag: 465642da06051a55630d39c899697b678f66a7f7
+            params:
+              DEPLOY_ENV: ((deploy_env))
```

Comment:
Out of curiosity, do you know why we need `DEPLOY_ENV`?

Reply:
I think I left this in by mistake!